src/main/target/VGOODRCF4/target.h

```diff
@@ -0,0 +1,195 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+ #pragma once
+
+//#define USE_TARGET_CONFIG
+#define TARGET_BOARD_IDENTIFIER	"VGF4"
+#define TARGET_MANUFACTURER_IDENTIFIER	"VGRC"
+#define USBD_PRODUCT_STRING	"VGOODRCF4"
+
+#define LED0_PIN                PC14
+#define LED1_PIN                PC15
+
+#define USE_BEEPER
+#define BEEPER_PIN              PC13
+#define BEEPER_INVERTED
+
+//#define ENABLE_DSHOT_DMAR       true//debug for checking
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+
+#define GYRO_1_ALIGN            CW180_DEG
+#define ACC_1_ALIGN             CW180_DEG
+
+#define USE_EXTI
+#define USE_GYRO_EXTI
+//#define MPU_INT_EXTI
+#define GYRO_1_EXTI_PIN      	PC3 
+
+#define USE_MPU_DATA_READY_SIGNAL
+//#define ENSURE_MPU_DATA_READY_IS_LOW
+
+#define USE_ACC
```

Comment:
This is duplicated
```
#define USE_ACC
#define USE_ACC_SPI_MPU6000
#define USE_GYRO
#define USE_GYRO_SPI_MPU6000
```

Reply:
Can't duplicate？Or What's wrong with duplicate

Reply:
Why duplicate? You will only make the code occupy more space

Reply:
I got it!It appeared twice like "#define USE_ACC",I will fix it.thanks!